plaidml2/core/ffi.cc

```diff
@@ -58,22 +61,21 @@ void plaidml_init(plaidml_error* err) {
           el::Loggers::setVerboseLevel(level);
         }
       }
+      IVLOG(1, "plaidml_init");
+      Settings::Instance()->load();
       auto ctx = GlobalContext::getContext();
       auto eventlog_str = vertexai::env::Get("PLAIDML_EVENTLOG_FILENAME");
-      if (ctx && eventlog_str.size()) {
+      if (eventlog_str.size()) {
         IVLOG(1, "Logging events to " << eventlog_str);
-        auto config = "{\"@type\": \"type.vertex.ai/vertexai.eventing.file.proto.EventLog\", \"filename\": \"" +
-                      eventlog_str + "\"}";
-        auto pconfig = vertexai::ParseConfig<google::protobuf::Any>(config);
-        auto eventlog = vertexai::AnyFactoryMap<vertexai::context::EventLog>::Instance()->MakeInstance(*ctx, pconfig);
+        EventLog e;
+        e.set_filename(eventlog_str);
+        auto eventlog = EventLogFactory().MakeTypedInstance(*ctx, e);
```

Comment:
I think we can directly create an instance of the `vertexai::eventing::file::EventLog` without having to go thru the `EventLogFactory` business.